Makefile

```diff
@@ -43,7 +44,7 @@ test-lib: lib
 # help: test-lib-with-fortran          - Build SmartRedis clients into a dynamic library with least permissive compiler settings
 .PHONY: test-lib-with-fortran
 test-lib-with-fortran: SHELL:=/bin/bash
-test-lib-with-fortran: LIB_BUILD_ARGS="-DWERROR=ON"
+test-lib-with-fortran: LIB_BUILD_ARGS="-DWERROR=ON -DCOVERAGE=ON"
```

Comment:
I'm not sure it makes sense to always test Fortran with code coverage enabled. For performance testing, we'll definitely want it turned off.

Reply:
Agreed, this has been removed from all the testers. Now this can be enabled by calling the target with the correct `CMAKE_ARGS`